Tests/test_dynamic_loading.py

```diff
@@ -82,6 +82,11 @@ def test_can_import_trigger(self):
         triggers = self.get_package_in_folder(self.trigger_dir)
         package_name = "trigger"
         for trigger in triggers:
+
+            # snowboy is only compiled for python2
```

Comment:
[Here ](https://github.com/kalliope-project/kalliope/blob/master/kalliope/trigger/snowboy/snowboydetect.py#L11)is the line where we select the right so file.